pennylane/measurements/mid_measure.py

```diff
@@ -28,8 +28,8 @@ def measure(wires: Wires, reset: Optional[bool] = False, postselect: Optional[in
     r"""Perform a mid-circuit measurement in the computational basis on the
     supplied qubit.
 
-    Measurement outcomes can be obtained and used to conditionally apply
-    operations.
+    Measurement outcomes can be used to conditionally apply operations and measurement
+    statistics can be gathered and returned by a quantum function.
```

Comment:
Returning statistics is advertised in other sections, like the quickstart, of the documentation.